### DISCUSSION TITLE
How to free the memory ?

### DISCUSSION THREAD

**Discussion by octalflow** [litehtml/litehtml#376] [Q&A]
doc = createFromString();
doc->draw((litehtml::uint_ptr)hdc, 0, 0, nullptr);
....
after that
How to free the memory ?

**Comment by tordex** [comment]
```createFromString``` returns ```std::shared_ptr```. So it free memory automatically or just do ```doc = nullptr```.

PS. Please use discussions for questions

**Comment by octalflow** [comment]
.
